src/map-interactions.react.js

```diff
@@ -126,12 +127,12 @@ export default class Interactions extends Component {
   }
 
   _getMousePos(event) {
-    const el = this.refs.container;
+    const el = this.container;
```

Comment:
Can we initialize these to `null` in the `constructor` above and follow convention of prefixing these with `_` to indicate they're supposed to be private? e.g.
```js
constructor(props) {
  ...
  this._container = null;
}
```